src/Site.js

```diff
@@ -46,12 +49,15 @@ const PROJECT_PLUGIN_FOLDER_NAME = '_markbind/plugins';
 const SITE_CONFIG_NAME = 'site.json';
 const SITE_DATA_NAME = 'siteData.json';
 const SITE_NAV_PATH = '_markbind/navigation/site-nav.md';
+const TOP_NAV_PATH = '_markbind/common/topNav.md';
```

Comment:
`topNav` -> `top-nav`? (to be more consistent with the file naming convention in general).